### ISSUE TITLE
CAN and UAVCAN at the same time

### ISSUE THREAD

**Issue by parakhm95** [mjs513/libuavcan-Teensy-3.x#1]
Thank you so much for investing your time on this and making it work for everyone. I used the uavcan to work with Pixhawk using a teensy and it worked very easily. Now I am trying to use the DUAL CAN bus of Teensy 3.6 to write a translation layer between a standard CAN and Pixhawk using your library. I have been trying using :

canDriver->getIface(1)->receive(recFrame,montime, utctime, canioflags)

but I am not getting anything. Can you please help me?

**Comment by mjs513** [comment]
A little lost here.  Not 100% sure what you are trying to do.

If you connect the CAN bus to pixhawk you can exchange messages.  So not sure what you are trying to do with the second bus?  Also where are you trying to make that change?

Think you want to use CAN0 for UAVCAN and then use CAN1 for reading other can devices?

**Comment by parakhm95** [comment]
> Think you want to use CAN0 for UAVCAN and then use CAN1 for reading other can devices?

Yes that is what I wanted to do. I was able to access one of the CAN interfaces and change the filter to achieve it. I think we can close this.
